IndicatorFields/incidentfield-Cymulate_Immediate_Threats_Status.json

```diff
@@ -0,0 +1,40 @@
+{
+	"associatedToAll": false,
+	"associatedTypes": [
```

Comment:
these fields are incident fields, move them under `IncidentFields` folder
Same for all the other fields